crpg.py

```diff
@@ -81,19 +83,21 @@ def generate(filename):
         node_mapping: dict[str, Node] = {}
         n_types = {
             "starting": Location,
-            "node": Node,
+            "node": Action,
```

Comment:
A node should be of type Node, if you want to add an option for action, you could add it as a separate key: value pair in the map